polyply/src/generate_templates.py

```diff
@@ -255,10 +255,11 @@ class GenerateTemplates(Processor):
     of each block in the volume attribute.
     """
     def __init__(self, *args, **kwargs):
-        super().__init__(*args, **kwargs)
+        super().__init__()
         self.templates = {}
         self.resnames = []
         self.volumes = {}
+        self.max_opt = kwargs["max_opt"]
```

Comment:
Add `max_opt` as argument, and pass `*args` and `**kwargs` to `super().__init__()` again

Reply:
but super doesn't take any args or kwargs? So what is the point?

Reply:
I just gave it a single max_opt argument